### DISCUSSION TITLE
FetchXML Builder - Loading of XrmToolBox.PluginsStore Error

### DISCUSSION THREAD

**Discussion by galactic-plane** [rappen/FetchXMLBuilder#988] [General]
Hi, not sure why this is happening, but when I click the tool I get "Could not load file or assembly 'XrmToolBox.PluginsStore, Version=1.2022.10.58, Culture=nutral, PublicKeyToken=null"...  This is a new installation of XrmToolBox (just downloaded, first run), went to store and downloaded and installed the FetchXML Builder and get that error when trying to use it.  Close app and reopen, still get the error.  Ran the XrmTooBox as an Admin, still get the error.  Thanks!

**Comment by suduko-tom** [comment]
I am getting the same problem with a new PC and installation

**Comment by LucasCXavier** [comment]
Hi guys,
I'm facing the same issue with a new PC and installation.

**Comment by rappen** [comment]
I'm sorry for these issues... I blame Tanguy 😉
I will release a new version of all my tools, unfortunately I'm quite busy these days. I'll try!
I'll close this issue, I assume that link solves it, for now.
https://jonasr.app/missing-xrmtoolbox-pluginsstore/

**Reply by suduko-tom** [reply]
Brilliant, thanks, worked like a dream.

**Comment by galactic-plane** [comment]
Thank you!
